scripts/v.import/v.import.py

```diff
@@ -208,8 +252,12 @@ def main():
     # create temp location from input without import
     grass.verbose(_("Creating temporary location for <%s>...") % OGRdatasource)
     try:
+        if checkGDALversion():
```

Comment:
The suggestions above should make this condition more readble in terms of what is the intend.